### PR TITLE
Added option to ignore input errors

### LINKED ISSUE #56: [FEATURE] Adding flags to ignore input errors
Adding flags to ignore input errors

### PR DESCRIPTION
**Pull request by hwangswan**
Resolve #56 

Added option to ignore input errors:
```
./main --input <input_csv> --ignore-input-error
```

Note that `error` without the `s`.

### CODE REVIEW

File: src/OutputHelper.cpp [hunk 1]
```diff
@@ -39,7 +39,7 @@ bool OutputHelper::isCSVOutput() {
  *
  * @param  int
  */
-void OutputHelper::setOutputFormat(int flag) {
+void OutputHelper::setOutputFlag(int flag) {
   _outputType |= flag;
```
Comment (trhgquan) [inline src/OutputHelper.cpp:43]: Shouldn't it be `_outputFlag` ?

**Review by trhgquan — CHANGES_REQUESTED** [review]
Missing readme.md, also something needs to be changed.

File: src/OutputHelper.cpp (hunk 1 above)
Comment (hwangswan) [reply src/OutputHelper.cpp:43]: The variable name is `_outputType` from the beginning.

Comment (trhgquan) [reply src/OutputHelper.cpp:43]: I know that, but it should be changed to `_outputFlag` for further maintenance.

Comment (hwangswan) [reply src/OutputHelper.cpp:43]: Gonna fix this soon!

**Review by trhgquan — APPROVED** [review]
Done.

**Pull request merged**